pkg/scheduler/scheduler.go

```diff
@@ -121,9 +126,7 @@ func (s *Scheduler) schedule(ctx context.Context) {
 		if admittedWorkloads.Has(workload.Key(w.Obj)) {
 			continue
 		}
-		if s.queues.RequeueWorkload(ctx, &w) {
-			log.V(2).Info("Workload re-queued", "queuedWorkload", klog.KObj(w.Obj), "queue", klog.KRef(w.Obj.Namespace, w.Obj.Spec.QueueName))
-		}
+		s.requeueAndUpdate(log, ctx, &w, "workload didn't fit")
```

Comment:
there are 2 scenarios here: one were the workload didn't fit at all, and one when it could have fit, but another workload got scheduled first (due to priority, or because it didn't require borrowing, etc).

But we can separate those as a follow up.

Reply:
In a followup, the nominate func should return two lists, the entries that fit and the ones that didn't, and for the latter list each entry should include a reason why it didn't that we can pass as the condition message.: https://github.com/kubernetes-sigs/kueue/pull/125#discussion_r830546251